src/utils/validation.js

```diff
@@ -1,8 +1,64 @@
 import HttpError from 'node-http-error';
 import isEmpty from 'lodash/isEmpty';
+import mongoose from 'mongoose';
 
 export const ValidationError = message => HttpError(400, message);
 
+// TODO(uladbohdan): to replace messages with error codes.
+
+// These are the drafts for validators.
+// TODO(uladbohdan): to refactor, to extend.
+
+// TODO(uladbohdan): to figure out nesting in errors object.
+
+const createArticleValidator = ({ body }, res, next) => {
+  const errors = {};
+  if (body.locales) {
+    Object.keys(body.locales).forEach(loc => {
+      ['title', 'subtitle', 'slug', 'text', 'locale'].forEach(field => {
+        if (!body.locales[loc][field]) {
+          errors[field] = 'must be presented';
+        }
+      });
+      if (loc !== body.locales[loc].locale) {
```

Comment:
how can this be possible?

Reply:
well something might go wrong in Frontend. It's better to check